src/main/webapp/script.js

```diff
@@ -104,20 +104,36 @@ function createParagraphElement(text) {
 // This function returns a message based on sentiment scores
 function getScoreResponse(score) {
   // Messages based on score ranges
-  //TODO: update messages to be something more readable
-  const messages = ["Negative", "Neutral", "Positive"];
+  //TODO: updated messages to a series of good/bad/netural messages. Make it a random selection.
+  const positiveMessages = [
+    "Liking the positive energy!!",
+    "That should make someone smile.",
+    "Killing them with kindness.",
```

Comment:
The use of killing here may not be appropriate?

Reply:
I think you're right. i'll update it to something more appropriate.

Reply:
Just updated it.